### PR TITLE
fix: event_id_counter not being used in subscribe_to_system_event()

### PR DESCRIPTION
**Pull request by juliansebline**
This PR fixes the issue #19

### CODE REVIEW

**Review by Gurgel100 — COMMENTED** [review]
Please fix the formatting.

File: msfs/src/sim_connect.rs [hunk 1]
```diff
@@ -468,18 +468,19 @@ impl<'a> SimConnect<'a> {
...
         &mut self,
-        event_id: sys::SIMCONNECT_CLIENT_EVENT_ID,
         system_event_name: &str,
-    ) -> Result<()> {
-        let system_event_name = std::ffi::CString::new(system_event_name).unwrap();
+    ) -> Result<sys::DWORD> {
+        let event_id = self.event_id_counter;
+        self.event_id_counter += 1;
         unsafe {
+            let system_event_name = std::ffi::CString::new(system_event_name).unwrap();
```
[the remark this answers is not in the document]
Comment (juliansebline) [reply msfs/src/sim_connect.rs:476]: Because it's done this way in `map_client_event_to_sim_event`

[the remark this answers is not in the document]
Comment (Gurgel100) [reply msfs/src/sim_connect.rs:476]: In my opinionen the less code in the `unsafe` block the better so could you move it out of there? Also in `map_client_event_to_sim_event`.

[the remark this answers is not in the document]
Comment (juliansebline) [reply msfs/src/sim_connect.rs:476]: Roger

[the remark this answers is not in the document]
Comment (juliansebline) [reply msfs/src/sim_connect.rs]: Done. I did it for another function too.

**Review by Gurgel100 — APPROVED** [review]
LGTM

**Comment by Gurgel100** [comment]
You still need to fix the formatting.

**Comment by juliansebline** [comment]
> You still need to fix the formatting.

Which formatting? You mean the version? If yes, I've tried but I don't know how it works.

**Comment by Gurgel100** [comment]
> > You still need to fix the formatting.
> 
> Which formatting? You mean the version? If yes, I've tried but I don't know how it works.

I mean the formatting of the code (see failing check).

**Comment by juliansebline** [comment]
> > > You still need to fix the formatting.
> > 
> > 
> > Which formatting? You mean the version? If yes, I've tried but I don't know how it works.
> 
> I mean the formatting of the code (see failing check).

I don't see the formatting problem, sorry. Can you show me or fix it? Please

**Pull request merged**